src/package.json

```diff
@@ -1,15 +1,10 @@
 {
-  "name": "src",
+  "name": "bird-brain",
```

Comment:
`"name"` is used by chromium when creating the appData folder for this app, visible from `console.log(nw.App.dataPath);`. So it's good to pick a unique, app-specific name.